docs/MSC1711_certificates_FAQ.md

```diff
@@ -132,7 +132,7 @@ your domain, you can simply route all traffic through the reverse proxy by
 updating the SRV record appropriately (or removing it, if the proxy listens on
 8448).
 
-See [reverse_proxy.md](reverse_proxy.md) for information on setting up a
+See [the reverse proxy page](reverse_proxy.md) for information on setting up a
```

Comment:
Would it make more sense to use "the reverse proxy documentation" for each of these? I feel like referring them as a "page" isn't really useful. (What if it is a different part of the same page?!)

Reply:
Works for me!